src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -59,6 +58,11 @@
 using System.Xml.Xsl;
 using Microsoft.CSharp.RuntimeBinder;
 
+// System.Text.Json is a .NET Core 3.0 specific library
```

Comment:
@ViktorHofer, please review the changes to S.R.Serialization.Formatters tests. I added a netcoreapp specific config.